desc/compute/_omnigenity.py

```diff
@@ -346,6 +347,100 @@ def _f_T(params, transforms, profiles, data, **kwargs):
     return data
 
 
+# TODO: change eta to be from [-1, +1] instead of [-pi/2, +pi/2]
+@register_compute_fun(
+    name="eta",
+    label="\\eta",
+    units="rad",
+    units_long="radians",
+    description="Intermediate omnigenity coordinate along field lines",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="rtz",
+    data=[],
+    parameterization="desc.magnetic_fields.OmnigenousField",
+)
+def _eta(params, transforms, profiles, data, **kwargs):
+    # theta is used as a placeholder for eta (angle along field lines)
+    data["eta"] = (transforms["grid"].nodes[:, 1] - jnp.pi) / 2
+    return data
+
+
+@register_compute_fun(
+    name="alpha",
+    label="\\alpha",
+    units="rad",
+    units_long="radians",
+    description="Field line label, defined on [0, 2pi)",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="rtz",
+    data=[],
+    parameterization="desc.magnetic_fields.OmnigenousField",
+)
+def _alpha(params, transforms, profiles, data, **kwargs):
+    # zeta is used as a placeholder for alpha (field line label)
+    data["alpha"] = transforms["grid"].nodes[:, 2] * transforms["grid"].NFP
+    return data
+
+
+@register_compute_fun(
+    name="h",
+    label="h = \\theta + (N / M) \\zeta",
+    units="rad",
+    units_long="radians",
+    description="Omnigenity symmetry angle",
+    dim=1,
+    params=["x_lmn"],
+    transforms={"omni": [[0, 0, 0]]},
+    profiles=[],
+    coordinates="rtz",
+    data=["rho", "alpha", "eta"],
+    parameterization="desc.magnetic_fields.OmnigenousField",
+)
+def _omni_angle(params, transforms, profiles, data, **kwargs):
+    nodes = jnp.array([data["rho"], data["eta"], data["alpha"]]).T
+    data["h"] = (
+        transforms["omni"].basis.evaluate(nodes) @ params["x_lmn"]
+        + 2 * data["eta"]
+        + jnp.pi
+    )
+    return data
+
+
+@register_compute_fun(
+    name="|B|_omni",
+    label="|\\mathbf{B}_{omni}|",
+    units="T",
+    units_long="Tesla",
+    description="Magnitude of omnigenous magnetic field",
+    dim=1,
+    params=["B_lm"],
+    transforms={"well": [[0, 0, 0]]},
+    profiles=[],
+    coordinates="rtz",
+    data=["eta"],
+    parameterization="desc.magnetic_fields.OmnigenousField",
+)
+def _B_well(params, transforms, profiles, data, **kwargs):
+    # reshaped to size (L_well, M_well)
+    B_lm = params["B_lm"].reshape((transforms["well"].basis.L + 1, -1))
+    # assuming single flux surface, so only take first row (single node)
+    B_input = (transforms["well"].matrices["direct1"][0][0][0] @ B_lm)[0, :]
```

Comment:
We can't because `B_lm` here is a 2D array. We are doing the transform for each spline know, so doing the matrix multiplication manually like this is more efficient.

Reply:
We can let JAX handle the vectorization, ie if you add something like
```python
@functool.partial(jnp.vectorize, signature="(n)->(k)")
```
 as a decorator to `Transform.transform`

Reply:
that would let it also work with fft transforms etc.